index.html

```diff
@@ -915,37 +1027,105 @@ <h4>Text</h4>
 
         <section>
           <h4>Text Language Source</h4>
-          <p>The <dfn>Text Language Source</dfn> property is an annotation indicating whether a <a>Text</a> object is
-            in the same language as the relevant part of the audio's
-            language (original), or if it is a representation in another language (translation):</p>
+          <p>The <dfn>Text Language Source</dfn> property is an annotation indicating the source language of
+            a <a>Text</a> object that represents content with an inherent language.
+            If it differs from the language of the <a>Text</a> object,
+            that implies that it is a representation in another language,
+            referred to as &quot;translation&quot;;
+            conversely if it is the same that implies that the language of the <a>Text</a> is <a>Original</a>.</p>
```

Comment:
This sentence seems duplicative of the sentence at line 1059.

Reply:
Yes, one is describing the data model, the other the syntactic representation. The syntactic representation may change when I change `daptm:langSrc` to be more like `xml:lang` in its inheritance model.

Reply:
No change needed, I think?

Reply:
This has actually been adjusted now - there is still some duplication, but it is hopefully clearer now that one is about the data model, the other about the XML representation.